src/peerstate.rs

```diff
@@ -427,42 +459,49 @@ impl Peerstate {
         &mut self,
         which_key: PeerstateKeyType,
         fingerprint: Fingerprint,
-        verified: PeerstateVerifiedStatus,
         verifier: String,
     ) -> Result<()> {
-        if verified == PeerstateVerifiedStatus::BidirectVerified {
```

Comment:
sidenote: this if can be dropped because all callers of `set_verified` pass in `PeerstateVerifiedStatus::BidirectVerified` anyway.